cli/src/address_lookup_table.rs

```diff
@@ -0,0 +1,832 @@
+                    SubCommand::with_name("deactivate-lookup-table")
+                        .about("Permanently deactivates a lookup table")
+                        .arg(
+                            Arg::with_name("lookup_table_address")
+                                .index(1)
+                                .value_name("LOOKUP_TABLE_ADDRESS")
+                                .takes_value(true)
+                                .required(true)
+                                .help("Address of the lookup table")
+                        )
+                        .arg(
+                            Arg::with_name("authority")
+                                .long("authority")
+                                .value_name("AUTHORITY_SIGNER")
+                                .takes_value(true)
+                                .validator(is_valid_signer)
+                                .help("Lookup table authority [default: the default configured keypair]")
+                        )
+                        .arg(
+                            Arg::with_name("bypass_warning")
+                                .long("bypass-warning")
+                                .takes_value(false)
+                                .help("Bypass the permanent lookup table deactivation warning"),
+                        ),
+                )
+                .subcommand(
+                    SubCommand::with_name("close-lookup-table")
+                        .about("Permanently closes a lookup table")
+                        .arg(
+                            Arg::with_name("lookup_table_address")
+                                .index(1)
+                                .value_name("LOOKUP_TABLE_ADDRESS")
+                                .takes_value(true)
+                                .required(true)
+                                .help("Address of the lookup table")
+                        )
+                        .arg(
+                            Arg::with_name("recipient")
+                                .long("recipient")
+                                .value_name("RECIPIENT_ADDRESS")
+                                .takes_value(true)
+                                .validator(is_pubkey)
+                                .help("Address of the recipient account to deposit the closed account's lamports [default: the default configured keypair]")
+                        )
+                        .arg(
+                            Arg::with_name("authority")
+                                .long("authority")
+                                .value_name("AUTHORITY_SIGNER")
+                                .takes_value(true)
+                                .validator(is_valid_signer)
+                                .help("Lookup table authority [default: the default configured keypair]")
+                        )
+                )
+                .subcommand(
+                    SubCommand::with_name("show-lookup-table")
```

Comment:
(Total fly-by based on your PR description)
Wdyt about making this just `get`? eg `solana address-lookup-table get <ADDRESS>`
Can probably remove "lookup-table" from the rest of the subcommands as well?

Reply:
Yeah they are pretty wordy, I'll shorten those up and switch to `get` as well, thanks for the suggestion!